test/cli/simple-package-rbi-gen/test.out

```diff
@@ -1,15 +1,7 @@
 No errors! Great job.
 # typed: true
 
-module ::Project
-end
-
-module ::Project::Foo
-end
-
 class ::Project::Foo::FooClass
-  extend ::T::Sig
   def dynamically_defined_method(); end
-  def initialize(value); end
```

Comment:
ooc, what causes these changes?

Reply:
The previous implementation was incorrect, this change reflects the actual minimal rbi. This really caught me off guard, but the previous implementation mangling the names of all the definitions explains why the test.out on master has an rbi that describes the whole packaged source.